docs/api/hooks/useActionData.md

```diff
@@ -21,7 +21,7 @@ changes are merged.
 
 [Reference Documentation ↗](https://api.reactrouter.com/v7/functions/react_router.useActionData.html)
 
-Returns the action data from the most recent POST navigation form submission or `undefined` if there hasn't been one.
+Returns the [`action`](../../start/framework/route-module#action) data from the most recent `POST` navigation form submission or `undefined` if there hasn't been one.
```

Comment:
I think our script should be able to auto-generate the link - try this format:

```suggestion
Returns the {@link action} data from the most recent `POST` navigation form submission or `undefined` if there hasn't been one.
```

Reply:
@brophdawg11 You're suggesting changes in the MD files here, the JSDoc is using `@link` wherever possible

Reply:
lol oops - lets back out the `.md` file changes from this PR and just keep the `hooks.tsx` changes